sdcm/utils/udf_scripts/lua_var_length_counter_accumulator.yaml

```diff
@@ -1,6 +1,6 @@
 # UDF script: accumulator function counting up the number of chars in a given column
 
-name: 'var_length_counter'
+name: 'var_length_counter_acc'
```

Comment:
how this change is related to the fullscan thread series?

Reply:
My bad, it must of made it's way in when I was working on the UDA/UDF stuff. Fixed - removed.